storage/sql_migrations/3_didweb.up.sql

```diff
@@ -0,0 +1,38 @@
+-- this table is used to store the did:web
+create table vdr_didweb
+(
+    -- did is the fully qualified did:web
+    did varchar(500) not null,
+    primary key (did)
+);
+
+-- this table is used to store the services for a did:web
+create table vdr_didweb_service
+(
+    -- id is the unique id of the service as it appears in the DID document.
+    id               varchar(255)  not null,
+    -- did references the containing did:web
+    did              varchar(255)  not null,
+    -- type is the service type
+    type             varchar(255)  not null,
+    -- service_endpoint is the service endpoint
+    service_endpoint varchar(2000) not null,
```

Comment:
I have a feeling this 2000 is going to haunt us. TEXT? or one from the BLOB family?

Reply:
BLOB is probably better, but I was a bit hestitant (or lazy) because I had to figure out what the column types in all the different RDBMS's is. But will change it to a binary type.